smart-contracts/contracts/CosmosBridge.sol

```diff
@@ -91,7 +94,11 @@ contract CosmosBridge is CosmosBridgeStorage, Oracle {
         address tokenAddress,
         uint256 amount,
         bool doublePeg,
-        uint128 nonce
+        uint128 nonce,
+        uint256 _networkDescriptor,
+        string memory tokenName,
+        string memory tokenSymbol,
+        uint8 tokenDecimals
```

Comment:
Do we really want to base our prophecyID off of the tokenName, tokenSymbol and tokenDecimals?

Reply:
The requirement for prophecyID just to make it unique. the minimum implementation is just including the cosmosSender and cosmosSenderSequence. other data items are optional

Reply:
one question, for not double peg case, if we need tokenName, tokenSymbol and tokenDecimals data in the claimData every time.

Reply:
@ElliotFriedman , we don't need them there, I removed them from the prophecyID.
@juniuszhou unfortunately, we do need to pass tokenName, tokenSymbol and tokenDecimals every time. There are alternatives, but they would demand a bigger refactory of the code - we would need to separate those fields from the basic claimData and use an overloaded function to deal with the variation. I'm not sure it's worth it, but discussing it in one of our meetings might be good.